types/node/index.d.ts

```diff
@@ -6331,6 +6331,7 @@ declare module "stream" {
 declare module "util" {
     export interface InspectOptions extends NodeJS.InspectOptions { }
     export function format(format: any, ...param: any[]): string;
+    export function formatWithOptions(inspectOptions: InspectOptions, format: any, ...param: any[]): string;
```

Comment:
While you can pass `any` since node seems to convert it to string anyway the docs specifically mention `string`. Maybe this is out of scope for this PR because this is also typed in `format`. Is there a reason to use `any`?

Reply:
Thanks!
No special reason. I just did a simple combine of `inspect` and `format` and forgot to take a closer look to them first. I corrected `formatWithOptions` but keep format as it is for now.